### PR TITLE
add examples, badges, and consistent readme language

### PR DESCRIPTION
**Pull request by jzelinskie**

### CODE REVIEW

File: examples/v0/check.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+from authzed.api.v0 import CheckRequest, Client, ObjectAndRelation, User
+from grpcutil import bearer_token_credentials
+
+
+emilia = User("blog/user", "emilia")
```
Comment (josephschorr) [inline examples/v0/check.py:5]: Have the prefix come from a variable and then add a comment indicating it needs to be replaced

Comment (jzelinskie) [reply examples/v0/check.py:4]: I think that just adds more noise to the example. The intent behind the examples is to let people see the API and not show them the most succinct or elegant usage of it.

Comment (josephschorr) [reply examples/v0/check.py:4]: As discussed, okay for now

**Review by josephschorr — APPROVED** [review]
LGTM

**Pull request merged**